docs/API.md

```diff
@@ -1081,15 +1124,38 @@ must have a value greater than 1000 - `SRTO_PEERLATENCY`.
 | -------------------- | ----- | ------- | ------ | ------ | -------- | ------ |
 | `SRTO_SNDSYN`        |       | post    | `bool` | true   | true     | false  |
 
-- **[GET or SET]** - Synchronous (blocking) send mode 
+- **[GET or SET]** - When true, sets blocking mode on writing function, when
+it's not ready to perform the operation. When false ("non-blocking mode"), the
+writing function will in this case report error `SRT_EASYNCSND` and return
+immediately.
+
+- On a connected socket or group this applies to a sending function
+(`srt_send` and others) and a situation when there's no free space in
+the sender buffer, caused by inability to send all the scheduled data over
+the network. Readiness for this operation can be tested by checking the
+`SRT_EPOLL_OUT` flag.
+
+- On a freshly created socket or group it will have no effect until the socket
+turns into a connected state.
+
+- On a listener socket it will be derived by the accepted socket or group,
+but will have no effect on the listener socket itself.
+
+- Note that this flag set on a group is for group only, as it concerns the
+exact sending operation being done on a group.
 
 ---
 
 | OptName               | Since | Binding | Type  | Units  | Default  | Range  |
 | --------------------- | ----- | ------- | ----- | ------ | -------- | ------ |
 | `SRTO_SNDTIMEO`       |       | post    | `int` | msecs  | -1       | -1..   |
 
-- **[GET or SET]** - Blocking mode sending timeout (-1: infinite)
+- **[GET or SET]** - limit the time up to which the sending operation will
+block (see `SRTO_SNDSYN` for details), so when this time is exceeded, it
+will behave as if in "non-blocking mode". The -1 value means no time limit.
+
+- Note that this flag set on a group is for group only, as it concerns the
+exact sending operation being done on a group.
```

Comment:
- Note that this flag is for a group only, as it concerns a
specific sending operation being done on a group.

Reply:
Same above.